vnext/Shared/OInstance.cpp

```diff
@@ -425,6 +425,23 @@ InstanceImpl::InstanceImpl(
           m_turboModuleRegistry,
           m_innerInstance->getJSCallInvoker());
     } else {
+#if defined(USE_V8)
+      std::unique_ptr<facebook::jsi::PreparedScriptStore> preparedScriptStore = nullptr;
+
+      char tempPath[MAX_PATH];
+      if (GetTempPathA(MAX_PATH, tempPath)) {
```

Comment:
>GetTempPathA [](http://example.com/codeflow?start=10&length=12)

This is not very good, and can break for non-ascii user names. Please use Unicode version.

Reply:
FWIW, this piece of code is copied from 20 lines above; it could probably use a refactor, but trying to make a least friction change for now.